Competitive_Programming/C++/Heap/FindingKthLargestElement.md

```diff
@@ -0,0 +1,197 @@
+# Heap in Competitive Programming
```

Comment:
Change the title as well. Here the main topic is to find the Kth Largest Element right..

Reply:
there are two topics : Kth Largest Element & Top K Frequent Elements